data/campaigns/Northern_Rebirth/scenarios/10a_Stolen_Gold.cfg

```diff
@@ -933,17 +933,17 @@
 
         [message]
             speaker=Himadrin
-            message= _ "Princess Eryssa sends you this gold along with the order to stay clear of any Northern Elf if you know what’s good for you."
+            message= _ "Princess Eryssa sends you this gold along with the order to stay clear of any northern elf if you know what’s good for you."
```

Comment:
Reminder to restore Northern Elf/Elves to capitals, here and subsequent lines.

Reply:
Yep.